subsys/bluetooth/controller/Kconfig

```diff
@@ -30,6 +30,13 @@ endchoice
 
 comment "BLE Controller configuration"
 
+config BLUETOOTH_CONTROLLER_TO_HOST_FC
+	bool "Controller to Host Flow Control"
+	depends on BLUETOOTH_CONN
+	default y
```

Comment:
Don't we also want 'default n if BLUETOOTH_HCI_HOST'? Since AFAIK combined builds don't have any advantage of this?